src/ui/components/card-detail/CardDetails.scss

```diff
@@ -0,0 +1,144 @@
+.card-details {
```

Comment:
@Sotatek-DukeVu @bao-sotatek  

The name of the folder should follow the convention and be `CardDetails` instead of `card-details`.